dojo/templates/dojo/custom_pdf_report.html

```diff
@@ -21,7 +21,7 @@
     <meta name="pdfkit-debug-javascript" content=""/>
     <meta name="pdfkit-quiet" content=""/>
     <title>{{ report_name }}</title>
-    <link href="{{ host }}{% static "bootswatch-dist/css/bootstrap.css" %}" rel="stylesheet">
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Can we pull this in via yarn, we try to keep all dependencies local.

Reply:
Sure, I'll tweak it.

Reply:
Resolved.